diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js
index feefc9b..b294d2f 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js
@@ -71,40 +71,7 @@ function LatestVersionSuggestionLabel({
 const getVersionMainDoc = version => version.docs.find(doc => doc.id === version.mainDocId);
 
 function DocVersionSuggestions() {
-  const {
-    siteConfig: {
-      title: siteTitle
-    }
-  } = (0, _useDocusaurusContext.default)();
-  const {
-    pluginId
-  } = (0, _useDocs.useActivePlugin)({
-    failfast: true
-  });
-  const {
-    savePreferredVersionName
-  } = (0, _themeCommon.useDocsPreferredVersion)(pluginId);
-  const activeVersion = (0, _useDocs.useActiveVersion)(pluginId);
-  const {
-    latestDocSuggestion,
-    latestVersionSuggestion
-  } = (0, _useDocs.useDocVersionSuggestions)(pluginId); // No suggestion to be made
-
-  if (!latestVersionSuggestion) {
-    return <></>;
-  } // try to link to same doc in latest version (not always possible)
-  // fallback to main doc of latest version
-
-
-  const latestVersionSuggestedDoc = latestDocSuggestion !== null && latestDocSuggestion !== void 0 ? latestDocSuggestion : getVersionMainDoc(latestVersionSuggestion);
-  return <div className="alert alert--warning margin-bottom--md" role="alert">
-      <div>
-        {activeVersion.name === 'current' ? <UnreleasedVersionLabel siteTitle={siteTitle} versionLabel={activeVersion.label} /> : <UnmaintainedVersionLabel siteTitle={siteTitle} versionLabel={activeVersion.label} />}
-      </div>
-      <div className="margin-top--md">
-        <LatestVersionSuggestionLabel versionLabel={latestVersionSuggestion.label} to={latestVersionSuggestedDoc.path} onClick={() => savePreferredVersionName(latestVersionSuggestion.name)} />
-      </div>
-    </div>;
+  return null;
 }
 
 var _default = DocVersionSuggestions;
