diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js
index bc8baa7..b12bebf 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/DocVersionSuggestions/index.js
@@ -18,58 +18,7 @@ const getVersionMainDoc = (version) =>
   version.docs.find((doc) => doc.id === version.mainDocId);
 
 function DocVersionSuggestions() {
-  const {
-    siteConfig: {title: siteTitle},
-  } = useDocusaurusContext();
-  const {pluginId} = useActivePlugin({
-    failfast: true,
-  });
-  const {savePreferredVersionName} = useDocsPreferredVersion(pluginId);
-  const activeVersion = useActiveVersion(pluginId);
-  const {
-    latestDocSuggestion,
-    latestVersionSuggestion,
-  } = useDocVersionSuggestions(pluginId); // No suggestion to be made
-
-  if (!latestVersionSuggestion) {
-    return <></>;
-  } // try to link to same doc in latest version (not always possible)
-  // fallback to main doc of latest version
-
-  const latestVersionSuggestedDoc =
-    latestDocSuggestion ?? getVersionMainDoc(latestVersionSuggestion);
-  return (
-    <div className="alert alert--warning margin-bottom--md" role="alert">
-      {
-        // TODO need refactoring
-        activeVersion.name === 'current' ? (
-          <div>
-            This is unreleased documentation for {siteTitle}{' '}
-            <strong>{activeVersion.label}</strong> version.
-          </div>
-        ) : (
-          <div>
-            This is documentation for {siteTitle}{' '}
-            <strong>{activeVersion.label}</strong>, which is no longer actively
-            maintained.
-          </div>
-        )
-      }
-      <div className="margin-top--md">
-        For up-to-date documentation, see the{' '}
-        <strong>
-          <Link
-            to={latestVersionSuggestedDoc.path}
-            onClick={() =>
-              savePreferredVersionName(latestVersionSuggestion.name)
-            }>
-            latest version
-          </Link>
-        </strong>{' '}
-        ({latestVersionSuggestion.label}).
-      </div>
-    </div>
-  );
+  return null
 }
 
 export default DocVersionSuggestions;
